Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -54,7 +54,7 @@ struct SelectTracks {
         status = 0;
       array<float, 2> dca;
       auto trackparvar0 = getTrackParCov(track);
-      trackparvar0.propagateParamToDCA(vtxXYZ, d_bz, &dca); // get impact parameters
+      trackparvar0.propagateParamToDCA(vtxXYZ, d_bz, &dca, 100.); // get impact parameters
```

Comment:
The dca will be assigned only if the propagation was successful (including condition that the straigh line (!) extrapolation is within ``maxD`` (i.e. 100 in this case).
So, one should always check that the returned result is true.
BTW, this is true for all track methods which can modify its state (just see if the method returns bool), there will be always some tracks which fail the operation.